README.md

```diff
@@ -16,3 +16,17 @@ For example, to build WASM for the [multi-memory](https://github.com/mdn/webasse
 ```sh
 wat2wasm --enable-multi-memory multi-memory.wat
 ```
+
+## Modifying the JS builtin examples
```

Comment:
Mind moving this into the `js-builtin-examples` dir as a README.md?

Reply:
Done!